integration_tests/backend/test_reads.py

```diff
@@ -496,29 +496,33 @@ def test_endpoint_node_artifact_get(self):
             assert sum(all_output_counts) == len(all_output_counts) - 1
             assert set(all_output_counts) == set([0, 1])
 
-    # def test_endpoint_node_artifact_result_content_get(self):
-    #     flow_id, n_runs = self.flows["flow_with_multiple_operators"]
-    #     flow = self.client.flow(flow_id)
-    #     workflow_resp = flow._get_workflow_resp()
-    #     dag_id = workflow_resp.workflow_dag_results[0].workflow_dag_id
-    #     dag_result_id = workflow_resp.workflow_dag_results[0].id
+    def test_endpoint_node_artifact_result_content_get(self):
+        flow_id, _ = self.flows["flow_with_multiple_operators"]
+        flow = self.client.flow(flow_id)
+        workflow_resp = flow._get_workflow_resp()
+        dag_id = workflow_resp.workflow_dag_results[0].workflow_dag_id
+        dag_result_id = workflow_resp.workflow_dag_results[0].id
 
-    #     dag_result_resp = globals.__GLOBAL_API_CLIENT__.get_workflow_dag_result(
-    #         flow_id,
-    #         dag_result_id,
-    #     )
-    #     artifact_ids = list(dag_result_resp.artifacts.keys())
-    #     artifact_id = str(artifact_ids[0])
-
-    #     resp = self.get_response(self.GET_NODE_ARTIFACT_RESULTS_TEMPLATE % (flow_id, dag_id, artifact_id)).json()
-    #     downstream_ids = [GetArtifactResultResponse(**result).id for result in resp]
-    #     for downstream_id in downstream_ids:
-    #         artifact_result_id = str(downstream_id)
-    #         resp = self.get_response(self.GET_NODE_ARTIFACT_RESULT_CONTENT_TEMPLATE % (flow_id, dag_id, artifact_id, artifact_result_id)).json()
-    #         # One of these should be successful (direct descendent of operator)
-    #         print(resp)
-    #     # TODO: Investigate output
-    #     # >> {"error":"Unexpected error reading DAG.\nQuery returned no rows."}
+        dag_result_resp = globals.__GLOBAL_API_CLIENT__.get_workflow_dag_result(
+            flow_id,
+            dag_result_id,
+        )
+        artifact_ids = list(dag_result_resp.artifacts.keys())
+        artifact_id = str(artifact_ids[0])
+
+        resp = self.get_response(
+            self.GET_NODE_ARTIFACT_RESULTS_TEMPLATE % (flow_id, dag_id, artifact_id)
+        ).json()
+        downstream_ids = [GetArtifactResultResponse(**result).id for result in resp]
+        for downstream_id in downstream_ids:
+            artifact_result_id = str(downstream_id)
+            resp = self.get_response(
+                self.GET_NODE_ARTIFACT_RESULT_CONTENT_TEMPLATE
```

Comment:
I've been checking the format of the response is correct by checking to see if I could convert them to the *Response class I create for each endpoint e.g. `result = GetNodeArtifactResponse(**resp)` does not error out. Would be useful if you could make a ContentResponse class which can be used for the other ContentGet routes as well